packages/core/src/payment/headless-payment-method.ts

```diff
@@ -0,0 +1,6 @@
+export default interface HeadlessPaymentMethod<T = any> {
```

Comment:
Can `any` type be changed to `unknown`?

Reply:
Would be better to use the `any` type in this case for matching `initializationData` type with `PaymentMethod` interface which also uses `any` for `initializationData`.